core/loadpoint_api.go

```diff
@@ -218,10 +218,7 @@ func (lp *Loadpoint) SetPriority(prio int) {
 	defer lp.Unlock()
 
 	lp.log.DEBUG.Println("set priority:", prio)
-
```

Comment:
Warum? Eine Änderung ist doch immer noch notwendig?

Reply:
Wenn wir mit `priority=0` über die `updateLoadpointHandler` Route kommen wird `setPriority` sonst nicht aufgerufen und wir speichern den neuen Wert nicht in die DB.
https://github.com/evcc-io/evcc/blob/fcbed8b8b2240cb8189a973fb7ca32db313f8c56/core/loadpoint/config.go#L56-L57

Reply:
Verstehe ich nicht. Wenn die Prio schon 0 ist muss ja auch nix gespeichert werden? Was ist hier das Szenario?

Reply:
Ich frage v.a. weil das bei absolut jedem anderen API auch so ist. Wenns hier nicht geht muss es einen tieferen Grund geben der an anderer Stelle auch zuschlagen würde?

Reply:
Wir mergen die Daten nicht. Der alte Wert bleibt sonst bestehen.

Schau gerne selbst rein. Es gibt einen Test dazu. Alternative Lösungen Welcome 🤗